pkgs/top-level/python-packages.nix

```diff
@@ -26423,4 +26423,75 @@ in modules // {
     };
   };
 
+  sigtools = buildPythonPackage rec {
+    name = "sigtools-${version}";
+    version = "1.1a3";
+
+    src = pkgs.fetchurl {
+      url = "https://pypi.python.org/packages/source/s/sigtools/${name}.tar.gz";
+      sha256 = "190w14vzbiyvxcl9jmyyimpahar5b0bq69v9iv7chi852yi71w6w";
+    };
+
+    # Tests fail with : AttributeError: 'module' object has no attribute 'test_forwards'
+    doCheck = false;
```

Comment:
Archives from PyPI indeed have the preference, but if tests are not included in the PyPI archive while they are on say GitHub, then that is enough reason to use that source. 

Tox is a bit tricky. With one tox.ini file you can test against multiple interpreter versions, and this won't work with Nix. In this situation that is not the case though.

By default the checkPhase runs `python setup.py test`. It's nicest if that just works.